util.mjs

```diff
@@ -0,0 +1,9 @@
+const cwd = `file://${process.cwd()}`
```

Comment:
Don't forget to add this to the `files` section in `package.json`

Reply:
Good catch, fixing.